templates/layout.html

```diff
@@ -78,10 +78,11 @@
                 Support us! <span class="caret"></span>
               </a>
               <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="supportGroupDrop">
+                <li><a href="https://opencollective.com/cdnjs" target="_blank">via Open Collective</a></li>
                 <li><a href="https://www.bountysource.com/teams/cdnjs" target="_blank">via Bountysource</a></li>
+                <li><a href="https://www.patreon.com/cdnjs" target="_blank">via Patreon</a></li>
                 <li><a href="https://liberapay.com/cdnjs/" target="_blank">via Liberapay</a></li>
                 <li><a href="https://tip4commit.com/github/cdnjs/cdnjs" target="_blank">via tip4commit</a></li>
-                <li><a href="https://twitter.com/cdnjs" target="_blank">Contact us!</a></li>
```

Comment:
Can we keep this to be open to discuss any other plans?

Reply:
Is there any reason it is located under support us, it felt like the wrong place for it really as we already link it in the footer where it is more appropriately placed? @PeterDaveHello

Reply:
It shows we are open to other solutions, it's not about Twitter or not